### PR TITLE
feat: Create E2E tests

### PR DESCRIPTION
**Pull request by juanjjaramillo**
## Description

Create script and documentation to run E2E tests

## Type of change

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] New feature / enhancement (non-breaking change which adds functionality)
- [ ] Security fix
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] Add changelog entry following the [contributing guide](../CONTRIBUTING.md#pull-requests)
- [x] Documentation has been updated
- [x] This change requires changes in testing:
  - [ ] unit tests
  - [x] E2E tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### enhancement
 - Create E2E resources Helm chart by @juanjjaramillo in [#377](https://github.com/newrelic/newrelic-infra-operator/pull/377)
+- Create E2E tests by @juanjjaramillo in [#378](https://github.com/newrelic/newrelic-infra-operator/pull/378)
```
Comment (svetlanabrennan) [inline CHANGELOG.md:14]: Can you move this to be under the ## Unreleased section please?

File: README.md [hunk 2]
```diff
@@ -149,6 +149,9 @@ Both installing the `newrelic-infra-operator` chart or spinning up the environme
 It is also possible to run such tests against any cluster you have access to by setting the environment variable
 `TEST_KUBECONFIG=/your/kube/config/path`. 
 
+E2E Testing (Metric NRQL queries):
```
Comment (svetlanabrennan) [inline README.md:152]: Can you please make this a heading like you did[ in this pr](https://github.com/newrelic/nri-kube-events/pull/342/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R102) so its' easier to find in the readme?

**Review by svetlanabrennan — APPROVED** [review]

**Comment by svetlanabrennan** [comment]
Just a few small changes but the rest looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (juanjjaramillo) [reply CHANGELOG.md:14]: Nice catch! Done

File: README.md (hunk 2 above)
Comment (juanjjaramillo) [reply README.md:152]: I'll do that in PR #379 since I changed the wording of that paragraph

**Pull request merged**